testing/cloud_layouts/script.sh

```diff
@@ -646,6 +650,8 @@ END_SCRIPT
       >&2 echo -n "  Deckhouse Pod not ready. Attempt $i/$testRunAttempts failed."
     fi
   done
+  >&2 echo -n "Fetch Deckhouse logs if error test ..."
+  $ssh_command -i "$ssh_private_key_path" $ssh_bastion "$ssh_user@$master_ip" sudo su -c /bin/bash > "$cwd/deckhouse.json.log" <<<"${testLog}"
```

Comment:
I suggest to no use $cwd for logging and put deckhouse.json.log into subdir of /tmp/ (e.g. /tmp/deckhouse), and upload it with this line:

```
            ${{ steps.setup.outputs.tmp-dir-path}}/deckhouse
```

Reply:
added /tmp/logs/